### DISCUSSION TITLE
EnvVar doesn't change value in airflow.cfg

### DISCUSSION THREAD

**Discussion by eugenelisevich** [apache/airflow#25666] [General]
### Apache Airflow version

Other Airflow 2 version

### What happened

Airflow image version 2.3.2-python3.10. I'm using Community Helm Chart for deployment the Airflow to the EKS and I need to enable AIRFLOW__WEBSERVER__ENABLE_PROXY_FIX, EnvVar present in the running container, but in airflow.cfg file inside the running container I see:
[webserver]
enable_proxy_fix = False

### What you think should happen instead

Should be 
[webserver]
enable_proxy_fix = True

### Operating System

Debian 11 (bullseye)

### Deployment

Other 3rd-party Helm chart

### Deployment details

Deployed from https://github.com/airflow-helm/charts Chart (8.6) using ArgoCD in EKS

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Please. please. don't open issues/bugs when you have questions. 

And if you have questions to the chart that is maintained outside of Airfflow community, you can raise your issues there.

If you read the docs of airflwo generally speaking if you set a variable it takes precedence but does not change the config. It would be rather bizzarre if by setting a variable Airflow would override the variable. But maybe it's done differently in the other chart. I think you should ask there.

**Reply by eugenelisevich** [reply]
Sorry for my mistake and thanks for your patience, but anyway I still have an issue with webserver behind proxy. I follow the HowTO https://airflow.apache.org/docs/apache-airflow/stable/howto/run-behind-proxy.html# but still webserver sends http instead of https in responce, should I open the new discussion or I can wait a responses here. Thank you Jarek.

**Reply by potiuk** [reply]
still - if you are using the chart from https://github.com/airflow-helm/charts - this is not the right place to ask those questions. Airlfow Chart is https://airflow.apache.org/docs/helm-chart/stable/index.html so you should ask your question where you took the chart from.
